lib/Controller/CatalogiController.php

```diff
@@ -122,6 +122,11 @@ public function create(ObjectService $objectService, DirectoryService $directory
 			}
 		}
 
+        // @todo dit is alleen omdat er een onredenlijke verplichting aan de database is toegeveoegd
+        if(array_key_exists('metadata',$data) === false){
+            $data['metadata'] = [];
+        }
+
```

Comment:
Hmm, I was able to remove this after updating the migrations. Did you try to rebuild the database before fixing this error? (The migrations contained errors that would crash certain database systems, hence we had to change existing migrations)

Reply:
I did, we kunnen hem er even inlaten met de todo en er laten op terucg komen